### PR TITLE
Add fityk

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/wojdyr/fityk/pull/40
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/wojdyr/fityk/pull/15 https://github.com/wojdyr/fityk/pull/40 https://github.com/wojdyr/fityk/issues/32

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build pl.nieto.fityk

File: fityk.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hfiguiere) [inline fityk.appdata.xml:1]: there is an appdata upstream, what's wrong with it?

File: pl.nieto.fityk.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: fityk
+rename-icon: fityk
+rename-desktop-file: fityk.desktop
+rename-appdata-file: fityk.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
```
Comment (hfiguiere) [inline pl.nieto.fityk.yaml:11]: this won't work since you force gtk2 with wxwidgets. gtk2 does NOT support wayland.

File: pl.nieto.fityk.yaml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        commit: 41967f1981d82d8495c0b27151a126bc35ae0d96
+        tag: v2.6.2
+    cleanup:
+      - /share/man
+      - /include
+
+  - name: swig
+    config-opts:
+      - --without-alllang
+      - --with-python=${FLATPAK_DEST}/bin/python3
```
Comment (hfiguiere) [inline pl.nieto.fityk.yaml:88]: python3 is in `/usr/bin`

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: fityk.appdata.xml (hunk 1 above)
Comment (hfiguiere) [reply fityk.appdata.xml:1]: I mean you could just add https://github.com/wojdyr/fityk/pull/40.patch

File: pl.nieto.fityk.yaml (hunk 2 above)
Comment (hfiguiere) [reply pl.nieto.fityk.yaml:11]: Building wxwidgets with gtk3 doesn't work?

Comment (Mailaender) [reply pl.nieto.fityk.yaml:11]: The author chose to make it fail the build https://github.com/wojdyr/fityk/commit/85ea545db65d7c6fbb94988b85f1e8cf1c9cbba5 but it seems the problems with Gtk3 are not that major as I thought.

**Comment by Mailaender** [comment]
bot, build pl.nieto.fityk

**Comment by nedrichards** [comment]
This did actually get merged correctly https://github.com/flathub/pl.nieto.fityk

**Pull request closed without merging**